e2e/async-esm-import/__tests__/test.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {jest} from '@jest/globals';
```

Comment:
```suggestion
```

Reply:
This import could not remove, because the next code will use the api `jest. isolateModulesAsync`